third_party/terraform/utils/sqladmin_operation.go

```diff
@@ -61,7 +61,24 @@ func (w *SqlAdminOperationWaiter) QueryOp() (interface{}, error) {
 		return nil, fmt.Errorf("Cannot query operation, service is nil.")
 	}
 
-	return w.Service.Operations.Get(w.Project, w.Op.Name).Do()
+	var op interface{}
+	err := retryTimeDuration(
+		func() error {
+			sqlOp, err := w.Service.Operations.Get(w.Project, w.Op.Name).Do()
+			if err != nil {
+				return err
+			} else {
```

Comment:
You can simplify this whole block by declaring `var err error` outside of the retry. You won't need the whole if/else and just `return err`

Reply:
Done- I think it's a little less clear what's going on, but avoids the aliasing problem I guess